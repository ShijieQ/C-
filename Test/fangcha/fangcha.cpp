#include<cmath>
#include<iostream>
#include<algorithm>

using namespace std;
const int maxn = 1e5+10;
double yizi[] = {0, 0, 8.99, 2.48, 1.59, 1.24, 1.05, 0.93, 0.84, 0.77, 0.72};

int main(){
    freopen("fangcha.in", "r", stdin);
    freopen("fangcha.out", "w", stdout);
    int n;
    cin>>n;
    double a[maxn];
    double average = 0;
    for(int i = 1; i <= n; i++){
        cin>>a[i];
        average += a[i];
    }
    average /= n;
    double ans = 0;
    for(int i = 1; i <= n; i++){
        ans += ((average - a[i])*(average - a[i]));
    }
    cout<<average<<endl;
    cout<<sqrt(ans/(n-1))<<endl;
    cout<<sqrt(ans/(n-1))*yizi[n]<<endl;
}