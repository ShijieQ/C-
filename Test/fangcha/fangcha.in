5
2.374 2.382 2.360 2.366 2.366
