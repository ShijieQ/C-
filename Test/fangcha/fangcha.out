2.3696
0.00853229
0.01058
